lang/zh.json

```
@@ -867,84 +867,84 @@
-      "condition": "Visit the following confection-themed locations: Chocolate World, Cutlery World, Sugar World (including the Sugary Depths), Candy Cane Fields, Ice Cream Dream, Cube of Sweets, Chocolate Tower, Dessert Fields (main area)"
+      "name": "糖糖！！",
+      "description": "对牙齿不好",
+      "condition": "抵达下列以甜点为主题的地点: 巧克力世界, 餐具世界, 砂糖世界（包括砂糖深渊）, 拐杖糖平原, 冰淇淋梦, 甜点方块, 巧克力塔, 甜点平原（主区域）"
     },
     "sunrise_drink": {
-      "name": "Sunrise Drink",
-      "description": "A warm-coloured beverage resembling the waking sky",
-      "condition": "Visit every bar in the game in the following locations: Japan Town, The Baddies Bar, Eyeball Archives, Cyber Bar, Cocktail Lounge, Aquamarine Cave, Spaceship, Oriental Pub, Water Reclamation Facility"
+      "name": "日出特饮",
+      "description": "宛如清晨之空的暖色饮品",
+      "condition": "抵达下列地点中的酒吧: 昭和路地, 坏蛋酒吧, 旋律绘本馆, 赛博酒吧, 鸡尾酒廊, 海蓝宝石洞穴, 宇宙船, 黑色东方酒吧, 净水设施"
     },
     "wavyup": {
-      "name": "Facing Up 2 Death",
-      "description": "She was filled with DETERMINATION",
-      "condition": "Complete the ↑V↑ minigame"
+      "name": "直面死亡",
+      "description": "她充满了 决 心",
+      "condition": "通关 ↑V↑ 小游戏"
     },
     "rune": {
-      "name": "Mana-infused Rune",
-      "description": "The arcane rune that connects the dream worlds",
-      "condition": "Visit the following locations: Intestines Maze, Digital Hand Hub, Urban Street Area, The Docks, Magnet Room (Zalgo's Room?), Mushroom World (White Mushroom Field), Crimson Labyrinth (Red Mountains), Color Cubes World, Octagonal Grid Hub (Broken Passage), Obentou World, Entropy Dungeon, Rainbow Towers"
+      "name": "注魔符文",
+      "description": "连接梦世界的神秘符文",
+      "condition": "抵达下列地点: 肉迷宫, 里青腕道路, 市街地, 雾之湖, 白色蘑菇世界, 溶岩山, 彩块世界, 八边形网格世界（中断的道路）, 便当世界, 熵之地牢, 彩虹之塔, 以及Zalgo事件的最终房间"
     },
     "poker": {
-      "name": "Gambling Zone",
-      "description": "You can only lose in this type of game",
-      "condition": "Enter the Poker Zone at the Despair Road"
+      "name": "赌博区域",
+      "description": "这种游戏十赌十输",
+      "condition": "进入拒绝之道的赌博区域"
     },
     "rainbow_ruins": {
-      "name": "Rainbow Brooch",
-      "condition": "Reach the Rainbow Ruins in the Bright Forest"
+      "name": "彩虹胸针",
+      "condition": "进入闪耀森林的彩虹遗迹"
     },
     "yukata": {
-      "name": "Yukata",
-      "condition": "Attempt to kill Yukata in The Invisible Maze until he sends you to an inescapable area"
+      "name": "师匠",
+      "condition": "反复尝试在隐形迷宫里锯掉师匠，直到他把你抓起来"
     },
     "blood_ghosts": {
-      "name": "Blood Ghosts",
-      "description": "You shouldn't have done that.",
-      "condition": "Activate the Blood Sacrifice event in Blood Wotrld"
+      "name": "鲜血幽灵",
+      "description": "不该这么做的...",
+      "condition": "在血生臭中触发血之献祭事件"
     },
     "scourge_of_guilt": {
-      "name": "Scourge of Guilt",
-      "description": "You monster...",
-      "condition": "Use the Chainsaw effect on the Perfume Bottle Ghost in the Shop Ruins"
+      "name": "悔恨之灾",
```

```suggestion
      "name": "疚之渊",
```
我感觉这里的Scourge应该趋向于“折磨”、“鞭挞”的意思。也就是说这个npc这么做以让流浪子坠入内疚的深渊/牢狱。